data_structures/Circular Queue.c

```diff
@@ -0,0 +1,75 @@
+#include<stdio.h>
+#define sizeQueue 10
+int totalElement;
+struct queue{
```

Comment:
Please format your full code according this code style conventions  
https://www.gnu.org/prep/standards/html_node/Formatting.html#Formatting
https://www.gnu.org/prep/standards/html_node/Syntactic-Conventions.html#Syntactic-Conventions
https://www.gnu.org/prep/standards/html_node/Writing-C.html